tomls/omnibus-base-mainnet-andromeda/perps/markets/sol.toml

```diff
@@ -21,7 +21,7 @@ sol_perps_endorsed_liquidator = "0x11233749514Ab8d00C0A5873DF7428b3db70030f"
 target = ["perpsFactory.PerpsMarketProxy"]
 fromCall.func = "owner"
 func = "createMarket"
-args = ["<%= settings.sol_perps_market_id %>", "Solana", "SOL"]
+args = ["<%= settings.sol_perps_market_id %>", "Sol", "SOL"]
```

Comment:
Same here. This will create new perps market. Better revert

Reply:
same here fixed in pr #3  so it has no effect to keep it